pydevd_plugins/extensions/pydevd_plugin_omegaconf.py

```diff
@@ -29,43 +34,59 @@ def get_dictionary(self, obj: Any) -> Any:
 
 
 class OmegaConfUserResolver(StrPresentationProvider):  # type: ignore
+    def __init__(self) -> None:
+        self.Node = find_mod_attr("omegaconf", "Node")
+        self.ValueNode = find_mod_attr("omegaconf", "ValueNode")
+        self.ListConfig = find_mod_attr("omegaconf", "ListConfig")
+        self.DictConfig = find_mod_attr("omegaconf", "DictConfig")
+        self.InterpolationResolutionError = find_mod_attr(
+            "omegaconf.errors", "InterpolationResolutionError"
+        )
+
     def can_provide(self, type_object: Any, type_name: str) -> bool:
-        Node = find_mod_attr("omegaconf", "Node")
-        return Node is not None and issubclass(type_object, Node)
+        return self.Node is not None and issubclass(type_object, self.Node)
 
     def resolve(self, obj: Any, attribute: Any) -> Any:
         if isinstance(obj, Sequence) and isinstance(attribute, str):
             attribute = int(attribute)
+
+        if isinstance(obj, self.Node):
```

Comment:
Minor:
```suggestion
        elif isinstance(obj, self.Node):
```

Reply:
no. the second if should happen regardless of the first.

Reply:
Oh, right, my bad.